index.html

```diff
@@ -45,9 +48,196 @@ <h2>Software Developer</h2>
                     <p>I can help you build a product , feature or website Look through some of my work and experience!
                         If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
                 </div>
+                <div class="hr-flex">
+                    <hr>
+                </div>
 
             </section>
+            <div>
+                <div class="box-flex lang-frame">
+
+                    <div class="box-size">
+                        <ul>
+                            <li>language</li>
+                            <li>javascript</li>
+                            <li>Ruby</li>
+                            <li>Html</li>
+                            <li>Css</li>
+                        </ul>
+                    </div>
+                    <div class="box-size">
+                        <ul>
+                            <li>frameworks</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSPec</li>
+                            <li>CapyBara</li>
+                        </ul>
+                    </div>
+                    <div class="box-size">
+                        <ul>
+                            <li>skills</li>
+                            <li>codekit</li>
+                            <li>github</li>
+                            <li>codepen</li>
+                            <li>gitlb</li>
+                        </ul>
+                    </div>
+
+                </div>
```

Comment:
- [ optional ] It will be nice to wrap technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, each of them in a `li`